routing_model/src/scenario_tests.rs

```diff
@@ -825,6 +825,32 @@ mod dst_tests {
             &AssertState::default(),
         );
     }
+
+    #[test]
+    fn test_local_event_offline() {
+        run_test(
+            "Get local event node detected offline",
+            &intial_state_old_elders(),
+            &[LocalEvent::NodeDetectedOffline(NODE_ELDER_130).to_event()],
+            &AssertState {
+                action_our_votes: vec![ParsecVote::Offline(NODE_ELDER_130)],
+                ..AssertState::default()
+            },
+        );
+    }
+
+    #[test]
+    fn test_local_event_online_again() {
+        run_test(
+            "Get local event node detected online again",
+            &intial_state_old_elders(),
+            &[LocalEvent::NodeDetectedBackOnline(NODE_ELDER_130).to_event()],
+            &AssertState {
+                action_our_votes: vec![ParsecVote::BackOnline(NODE_ELDER_130)],
+                ..AssertState::default()
+            },
+        );
+    }
```

Comment:
Missing a test for the parsec consensus for offline.
Probably also want one for consensus offline, and then checkElder happens (Maybe with the check elder tests).